src/Eppie.App/Eppie.App.ViewModels/LocalAIAgentSettingsPageViewModel.cs

```diff
@@ -527,8 +539,20 @@ private async Task UpdateAIAgentsListAsync()
                 AIAgentsList.SetItems(filteredAgents);
             }
 
+            var noneAgent = new LocalAIAgent() { Name = GetLocalizedString("NoneText") };
+            AIAgentsList.Insert(0, noneAgent);
+
             PreprocessorAIAgent = AIAgentsList.FirstOrDefault(agent => agent.Id == AgentSettingsModel.CurrentAgent.PreProcessorAgentId);
+            if (PreprocessorAIAgent == null)
```

Comment:
may be `PreprocessorAIAgent` rename to `PreProcessorAIAgent` like everywhere else